site/develop/index.md

```diff
@@ -123,3 +136,4 @@ We value all contributions that help to build a vibrant community, not just code
 You can contribute by testing the code, helping verify a release,
 writing documentation or the web site,
 or just by answering questions on the list.
+
```

Comment:
Is this blank line necessary?

Reply:
I removed by accident before, so I add it back now.